### PR TITLE
Added system.properties, trying to fix heroku deploy

### PR DESCRIPTION
**Pull request by gyapeee**

### CODE REVIEW

File: system.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+java.runtime.version=11
```
Comment (zoltansolt) [inline system.properties:1]: No newline at the end - error

**Review by zoltansolt — CHANGES_REQUESTED** [review]
No newline at the end of file

File: system.properties (hunk 1 above)
Comment (gyapeee) [reply system.properties]: fixed missing new line at eof

**Review by gyapeee — COMMENTED** [review]
new line at the end of prop file is added and app can be run

**Review by zoltansolt — APPROVED** [review]

**Pull request merged**